index.html

```diff
@@ -1,47 +1,264 @@
+          <div class="two">
+            <h1 class="project-title">Multi-Post Stories</h1>
+          </div>
+          <div class="three">
+            <div class="tags">
+              <span class="canopy-color">Canopy</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>Back End Dev</span>
+              <img class="rectangle" src="images/Counter.png" alt="counter">
+              <span>2015</span></div>
+          </div>
+          <div class="four">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div class="five">
+            <div class="tags2">
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">html</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">css</p>
+                </div>
+              </div>
+              <div class="lang">
+                <div class="lang-block">
+                  <p class="lang-text">javaScript</p>
+                </div>
+              </div>
+            </div>
+            <div class="six"><button type="button" class="card-button">See Project</button></div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <h1 class="project-title">About</h1>
+      <h1 class="project-title">Myself</h1>
+
+      <p class="myself">
+        Hello I’m a software developer!
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience!
+        If you like what you see and have a
+        project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="met">LET'S CONNECT</h3> <br>
+      <ul class="social-media-icons">
         <li>
-          <a href="#"><img src="icons/twitter.svg" alt="twitter" id="twitter"></a>
+          <a href="#"><img src="icons/twitter.svg" alt="twitter" id="twitter2"></a>
         </li>
         <li>
-          <a href="#"><img src="icons/Linkedin.svg" alt="linkedin" id="linkedin"></a>
+          <a href="#"><img src="icons/Linkedin.svg" alt="linkedin" id="linkedin2"></a>
         </li>
         <li>
-          <a href="#"><img src="icons/medium.svg" alt="medium" id="medium"></a>
+          <a href="#"><img src="icons/medium.svg" alt="medium" id="medium2"></a>
         </li>
         <li>
-          <a href="#"><img src="icons/github.svg" alt="github" id="github"></a>
+          <a href="#"><img src="icons/github.svg" alt="github" id="github2"></a>
         </li>
         <li>
-          <a href="#"><img src="icons/v-sign.svg" alt="V-sign" id="V-sign"></a>
+          <a href="#"><img src="icons/v-sign.svg" alt="V-sign" id="V-sign2"></a>
         </li>
       </ul>
-
+      <button type="button" class="card-button">
+        Get my resume
+      </button>
```

Comment:
- [ ] There is a missing part in this section. Please implement this part also.
While implementing this section be sure you followed below guidelines :arrow_down: 

![Screenshot 2021-06-06 at 00-43-08 Figma](https://user-images.githubusercontent.com/1642014/120906539-e837d580-c662-11eb-85d8-42ac933c3ec2.png)